package.json

```diff
@@ -42,6 +42,7 @@
     "@fontsource/inter": "4.5.11",
     "@formiz/core": "1.8.1",
     "@formiz/validations": "1.0.0",
+    "@formkit/auto-animate": "^1.0.0-beta.3",
```

Comment:
Is it possible to remove the `^` like the other dependencies? This way, we tell `yarn` that we are sure to use the specified version and not another semantic version that might match. From our experience, this is great to avoid bugs.

Reply:
I have removed ^, and I think auto-animate is still under development so they have not yet released a stable version.